tests/src/test/scala/common/Wsk.scala

```diff
@@ -71,6 +71,8 @@ import whisk.utils.retry
 
 case class WskProps(
     authKey: String = WhiskProperties.readAuthKey(WhiskProperties.getAuthFileForTesting),
+    cert: String = WhiskProperties.getProperty("openwhisk.home") + "/ansible/roles/nginx/files/openwhisk-client-cert.pem",
+    key: String = WhiskProperties.getProperty("openwhisk.home") + "/ansible/roles/nginx/files/openwhisk-client-key.pem",
```

Comment:
@markusthoemmes 
`WhiskProperties.whiskHome` can't be accessed from `Wsk.scala`, because it is private `private static final String whiskHome;`